accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/helper/QueryHelper.java

```diff
@@ -143,14 +145,9 @@ static boolean hasQueryIndexForACLs(final Session session) throws RepositoryExce
         Row row = queryResult.getRows().nextRow();
         // inspired by https://github.com/apache/jackrabbit-oak/blob/cc8adb42d89bc4625138a62ab074e7794a4d39ab/oak-jcr/src/test/java/org/apache/jackrabbit/oak/jcr/query/QueryTest.java#L1092
         String plan = row.getValue("plan").getString();
-        String costJson = plan.substring(plan.lastIndexOf('{'));
-        
-        // use jackson for JSON parsing
-        ObjectMapper mapper = new ObjectMapper();
+        String costJsonStr = plan.substring(plan.lastIndexOf('{'));
```

Comment:
Probably we should log the JSON (on debug/verbose level) in case it changes in the future, so that we can more easily come up with a fix.